common/build/eslint-config-fluid/minimal.js

```diff
@@ -57,13 +57,20 @@ module.exports = {
 		"unused-imports",
 		// Plugin documentation: https://www.npmjs.com/package/eslint-plugin-unicorn
 		"unicorn",
+		// Custom ESLint rules
+		"fluid-custom-rules",
```

Comment:
I suggest using fluid-internal instead of fluid-custom-rules.

Reply:
Name changed.